lib/api/docl.exs

```diff
@@ -7,53 +8,55 @@ defmodule Alchemist.API.Docl do
   import IEx.Helpers, warn: false
 
   alias Alchemist.Helpers.ModuleInfo
+  alias Alchemist.Helpers.CaptureIO
 
-  def request(args) do
-    Application.put_env(:iex, :colors, [enabled: true])
-
+  def request(args, device) do
     args
     |> normalize
-    |> process
+    |> process(device)
 
-    IO.puts "END-OF-DOCL"
+    IO.puts device, "END, func_puts-OF-DOCL"
```

Comment:
@slashmili I guess this one is wrong? Shouldn't it be `IO.puts device, "END-OF-DOCL"` instead?